pkg/network/node/ovs/ovs.go

```diff
@@ -424,9 +440,20 @@ func (tx *ovsExecTx) DeleteFlows(flow string, args ...interface{}) {
 }
 
 func (tx *ovsExecTx) Commit() error {
-	err := tx.ovsif.bundle(tx.flows)
-
-	// Reset flow context
-	tx.flows = []string{}
-	return err
+	defer func() {
+		tx.flows = []string{}
+	}()
+	backoff := utilwait.Backoff{
+		Duration: 500 * time.Millisecond,
+		Factor:   1.25,
+		Steps:    RETRIES,
+	}
+	return utilwait.ExponentialBackoff(backoff, func() (bool, error) {
+		err := tx.ovsif.bundle(tx.flows)
+		if err == nil {
+			return true, nil
+		}
+		metrics.OVSRetries.Inc()
```

Comment:
We want to catch the number of failed transactions, right? So this metric won't be right... it should only be incremented on the second (and further) retries.

Reply:
OK, not sure I follow on these concepts. But I take it you're saying: "have a counter incremented for every failed OVS commit, and have a go-routine sync the metric `OVSRetries` every second with that counter value", a bit like we're doing for `UpdateOVSMetrics`...?

Reply:
Oh, I'm a dummy; this is after the return.

However, the metric is confusing; it's the number of failed OVS operations, not the number of retries. Can you update the metric to reflect that? And, while you're at it, add one for OVS successes?

Reply:
I changed the name to `OVSFailedOperations`, and added `OVSSuccessfulOperations`